massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -511,8 +511,17 @@ impl ProtocolWorker {
                 self.checked_operations
                     .extend(operation_ids.iter().copied());
 
-                // Announce operations to active nodes not knowing about it.
                 let to_announce: Vec<OperationId> = operation_ids.iter().copied().collect();
+                // Broadcast operations to active sender(channel) subscribers.
+                if self.config.broadcast_enabled {
```

Comment:
In this function we receive operationids from the others nodes to let us know them and we ask their content if we don't already have them later on on the code.
So here you will fetch in the storage only ops that we already have and so has already been sent to the WS when the content has been retrieved.

Now that you want to broadcast all the operations you can retrieve the information for the `operation_pool` `add_operations` function ([code](https://github.com/massalabs/massa/blob/main/massa-pool-worker/src/operation_pool.rs#L101)). It's called by the `node_operations_from_node` where you had your previous send and the API when someone use `send_operations`

Reply:
If I understand, we have to options:

1. Move the broadcast to add_operations in massa-pool-worker(I prefer this)
2. Broadcast from Public API.

Reply:
Done